### DISCUSSION TITLE
Art of War skill book completely bugged

### DISCUSSION THREAD

**Discussion by Myrik25** [FakeFishGames/Barotrauma#13898] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Since the last patch, the books are completely unstacked and if you click on them while in character's possession it disappears.

### Reproduction steps

Load a game, buy Art of War skill increase book, try to use them.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13857
